base/applications/regedit/lang/pt-PT.rc

```diff
@@ -197,9 +197,9 @@ BEGIN
     END
     POPUP ""
     BEGIN
-        MENUITEM "C&ortar", ID_HEXEDIT_CUT
-        MENUITEM "&Copiar", ID_HEXEDIT_COPY //FIXME: accelerator collision &C
-        MENUITEM "&Colar", ID_HEXEDIT_PASTE  //FIXME: accelerator collision &C
+        MENUITEM "Cor&tar", ID_HEXEDIT_CUT
```

Comment:
`MENUITEM "Cor&tar", ID_HEXEDIT_CUT` that you cahnged here, will now collide with
`        MENUITEM "Seleccionar &Tudo", ID_HEXEDIT_SELECT_ALL` at line 205